### ISSUE TITLE
Timeline on supporting later RN versions?

### ISSUE THREAD

**Issue by kirakik** [wix/react-native-repackager#4]
Hello,

I was just wondering if there was a timeline on when later versions of react native will be supported?

Thank you

**Comment by Gabrn** [comment]
Would like to know as well :)

**Comment by gmisa** [comment]
same here.  there is nothing out there that supports react-native e2e mocks.  waiting on this.

**Comment by zcsongor** [comment]
+1

**Comment by DanielZlotin** [comment]
I'll add support for RN51 soon, though you should know I don't intend to keep maintaining this project.
This project is a giant hack until [this PR is merged](https://github.com/facebook/react-native/pull/16948).
What we do for latest RN version here at Wix is we are starting to maintain our own fork of react-native, to avoid waiting for PRs like this to be merged.

**Comment by DanielZlotin** [comment]
@kirakik done, version `0.51.x`

**Comment by ilonashub** [comment]
hey... I know it's not maintainable... but just to give it a shot -  any chance of supporting newer RN versions? 😇

**Comment by DanielZlotin** [comment]
This project is no longer needed as of `react-native@0.55.x`, so will not be updated and maintained.

**Comment by ilonashub** [comment]
@DanielZlotin great thanks!
